src/Sylius/Component/Resource/Provider/LocaleProvider.php

```diff
@@ -51,4 +58,17 @@ public function getFallbackLocale()
     {
         return $this->fallbackLocale;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefaultLocale()
+    {
+        if(null === $this->defaultLocale) {
+
```

Comment:
Anyway, we can shorten it to `return $this->defaultLocale ?: $this->fallbackLocale`